lang/en.lyaml

```diff
@@ -275,6 +275,13 @@ en:
               success:
                 update: "Your schema has been updated in account \"{{ accountId }}\""
                 viewAtUrl: "Schema can be viewed at {{ url }}"
+    doctor:
+      describe: "The doctor is in"
```

Comment:
😆 placeholder?

One suggestion is: "Run diagnostics to detect common CLI issues"

Reply:
Yeah, all the copy is awaiting input from UX 😄 

I should have mentioned that